contributing.md

```diff
@@ -0,0 +1,80 @@
+> # Meraki Frontend Repository
+
+This repository contains the source code for the Meraki Frontend. The Meraki Frontend is a web application that allows users to learn python , spoken english and other things.
+
+The Deployed Links For Meraki.
+
+[Meraki Production ]("https://merakilearn.org")
+
+[Meraki Development ]("https://bhanwari-devi.vercel.app")
+
+> ### Tech Stack
+
+- React
+- Redux
+- [MUI](https://mui.com/)
+- nodeJs (V14)
+
+> # Installation and Setup Guide
+
+1. Fork the repository
+2. create a Clone of the repository
+3. Make sure you have nodejs installed on your system with Version 14
+4. Run `npm install` to install all the dependencies .
+5. create a new .env file in root and add this code to it.
+
+   ```.env
+    REACT_APP_GOOGLE_CLIENT_ID = ''
+    REACT_APP_MERAKI_URL = 'https://api.merakilearn.org'
+    REACT_APP_CHANAKYA_BASE_URL = 'https://join.navgurukul.org/api/'
+    REACT_APP_ADMISSIONS_URL = 'https://admissions.navgurukul.org/'
+    REACT_APP_API_KEY = ''
+    REACT_APP_authDomain = ''
+    REACT_APP_projectId = ''
+    REACT_APP_storageBucket =''
+    REACT_APP_messagingSenderId = ''
+    REACT_APP_appId = ''
+   ```
+
+6. Execute This command
+   ```git
+      git checkout -b <your-branch-name>
```

Comment:
Few points here:

1. Should we tell the contributor to make sure they have Git Bash installed first? Maybe this is obvious for more experienced developers, but we should probably expect that we'll have some people who want to contribute who haven't used the Git command line before.
2. By default, this will create the branch off `main` whereas we're asking them to base it off the `dev` branch. So we should probably recommend that they enter `git checkout dev` first to switch to that branch if they're not already on it.
3. We might want to mention that if they clone the repository and a branch is added later on the NG repository that they want to use, they can enter the following:
```
# first time only to alias the NG repository
git remote add upstream https://github.com/navgurukul/bhanwari-devi.git

# downloads the files from the branch 
git fetch upstream <branch-you-want-to-start-from>

# switches to that branch
git checkout <branch-you-want-to-start-from>
```
4. Maybe we want to recommend that they also download [GitHub Desktop](https://desktop.github.com/) so they can avoid working with the command line for normal use cases?